src/components/AppFooter/index.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+import type {BuildInformation} from '../../containers/App/types';
```

Comment:
types of props should be defined per component as in: https://flow.org/en/docs/react/components/#toc-class-components

```javascript
 type Props = {
      frontEndBuldNumber: number,
     deploymentEnvironment: string,
 };

const AppFooter = ({
   deploymentEnvironment,
   frontEndBuildNumber,
 }: Props) => <Well>
   <p>
     Front-End Version {deploymentEnvironment} {frontEndBuldNumber}
   </p>
   <p>Back-End Version (get from the api) </p>
 </Well>
```